src/nswcaselaw/decision.py

```diff
@@ -407,12 +430,15 @@ def _scrape_judgment(self):
         paragraphs = []
         for child in body.children:
             if child.name == "h2":
-                paragraphs.append("##" + self._strings(child))
+                paragraphs.append(f"## {self._strings(child)}")
             elif child.name == "ol":
-                n = int(child["start"])
-                for li in child.find_all("li"):
-                    paragraphs.append(str(n) + " " + self._strings(li))
-                    n += 1
+                try:
+                    n = int(child["start"])
+                    for li in child.find_all("li"):
+                        paragraphs.append(f"{n} {self._strings(li)}")
+                        n += 1
+                except KeyError:
+                    pass
```

Comment:
I guess this was what was breaking the court of appeal HTML?

Reply:
This was one situation it was breaking, another was that SC pages didn't have Decision under appeal, and the logic for reading <dd> tags after <dt> didn't account for that